definitions/tools/biscuit_pileup.cwl

```diff
@@ -3,22 +3,30 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Biscuit pileup"
-baseCommand: ["/usr/bin/biscuit", "pileup"]
+baseCommand: ["/bin/bash", "helper.sh"]
 stdout: pileup.vcf.gz
 requirements:
     - class: ShellCommandRequirement
```

Comment:
Should be able to eliminate this requirement now (since we're calling a shell script ourselves instead!)